packages/components/src/templates/next/components/internal/Navbar/Navbar.tsx

```diff
@@ -43,6 +44,19 @@ export const Navbar = ({
         assetsBaseUrl: site.assetsBaseUrl,
         lazyLoading: false, // will always be above the fold
       }}
+      callToAction={
```

Comment:
nit: prefer `&&` over ternary here cos both end up with `undefined`, which makes the `&&` easier to read